unidist/config/backends/common/envvars.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Config entities which can be used for unidist common behavior tuning."""
+
+from packaging import version
+
+from unidist.config.parameter import EnvironmentVariable
+
+
+class Backend(EnvironmentVariable, type=str):
+    """Distribution backend to run queries by."""
+
+    varname = "UNIDIST_BACKEND"
+    choices = ("Ray", "Dask", "Mpi", "Multiprocessing", "Python")
```

Comment:
Because now we normalize a passed string from users. That is intentionally done to provide users flexibility as much as possible in setting the backend, i.e. they can pass `mpi`, `MPI`, `Mpi` and that will work. I think we can use string names for backends in the code as it is now without any affect to the rest entities you mentioned.